packages/manager/src/features/Databases/DatabaseDetail/index.tsx

```diff
@@ -81,6 +96,24 @@ export const DatabaseDetail: React.FC = () => {
     history.push(tabs[index].routeName);
   };
 
+  const handleSubmitLabelChange = (newLabel: string) => {
+    return updateDatabase({ allow_list: database.allow_list, label: newLabel })
```

Comment:
Wow that's unfortunate. I reported this as an API bug. `allow_list` should be optional.

```suggestion
  const handleSubmitLabelChange = (newLabel: string) => {
    // @TODO Update this to only send the label when the API supports it
    return updateDatabase({ allow_list: database.allow_list, label: newLabel })
```

Can we add some kind of comment to remind ourselves to update this at a later date when the API supports an optional `allow_list`?